src/pm.c

```diff
@@ -0,0 +1,472 @@
+#include <stdarg.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+
+#define PM_COUNTER	"counter"
+#define PM_GAUGE	"gauge"
+#define PM_HISTOGRAM	"histogram"
+
+#define PM_LEN			128
```

Comment:
nit: huh is there some weird spacing here and below? with these const values - seems like a difference in # tabs or spaces?

Reply:
github doesn't align things well, but in the code they are:
![image](https://user-images.githubusercontent.com/762345/82577635-4168c000-9b59-11ea-99cf-21508ecfffe7.png)